tasks/31_reverse-string-in-range/reverse-string-in-range.js

```diff
@@ -0,0 +1,18 @@
+
+
+export function reverseStringInRange(str, array) {
+    let ind1 = array[0];
+    let ind2 = array[1];
```

Comment:
ind1 and ind2 are never changed so it would be good for them to be defined as consts
also this syntax can be used to make it more compact:
`const [ind1, ind2] = array`